backend/db/models/booking.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Booking extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      Booking.belongsTo(
```

Comment:
dont forget to define foreign keys, sequelize does not always generate the correct ones.